docker-for-windows/troubleshoot.md

```diff
@@ -368,16 +353,16 @@ Here are some steps to take if you encounter similar problems:
 5.  If previous steps fail to solve the problems, follow steps on the [Cleanup
     README](https://github.com/Microsoft/Virtualization-Documentation/blob/master/windows-server-container-tools/CleanupContainerHostNetworking/README.md).
 
-    > Read full description before you run Windows cleanup script
+    > Read the full description before you run the Windows cleanup script.
     >
     >The cleanup command has two flags, `-Cleanup` and
     >`-ForceDeleteAllSwitches`. Read the whole page before running any scripts,
     >especially warnings about `-ForceDeleteAllSwitches`. {: .warning}
 
-### Windows containers and Windows Server 2016
+### Windows containers and Windows Server
 
 Docker Desktop is not supported on Windows Server 2016, instead you can use
-[Docker Enterprise Basic Edition](/ee/index.md) at no aditional cost.
+[Docker Enterprise Basic Edition](/ee/index.md) at no additional cost.
```

Comment:
semicolon rather than comma before "instead"

Reply:
Fixed in the review commit.